src/controllers/update/overseas.entity.query.controller.ts

```diff
@@ -0,0 +1,34 @@
+import { NextFunction, Request, Response } from "express";
+
+import { logger } from "../../utils/logger";
+import * as config from "../../config";
+import { getApplicationData, setExtraData } from "../../utils/application.data";
+import { OeNumberKey } from "../../model/data.types.model";
+
+export const get = (req: Request, res: Response, next: NextFunction) => {
+  try {
+    logger.debugRequest(req, `GET ${config.OVERSEAS_ENTITY_QUERY_PAGE}`);
+
+    return res.render(config.OVERSEAS_ENTITY_QUERY_PAGE, {
+      backLinkUrl: config.UPDATE_LANDING_PAGE_URL,
```

Comment:
Sorry probably I missed something, I can not see the landing page template for the update page.

Reply:
@mouhajer-ch , the landing page template for Update has still not been finalised

Reply:
I don't think there will be a view for the update landing page ...
We've followed the existing pattern for the register journey, where there is a landing controller with no associated view which just redirects to the first page of the journey `(LANDING_PAGE_URL=/register-an-overseas-entity/sold-land-filter?start=0)`.

So here in this scenario we have the update landing page redirect to `UPDATE_LANDING_PAGE_URL=/update-an-overseas-entity/overseas-entity-query`.

I presume this pattern is so it is relatively easy to change/re-order the first page of the journey?

Reply:
As I said, probably better if we remove the backlink block. When we do not know better do not add it